nixos/modules/services/web-apps/onlyoffice.nix

```diff
@@ -145,7 +145,7 @@ in
               '';
             "~* ^(\\/cache\\/files.*)(\\/.*)".extraConfig = ''
               alias /var/lib/onlyoffice/documentserver/App_Data$1;
-              add_header Content-Disposition "attachment; filename*=UTF-8''$arg_filename";
+              more_set_headers Content-Disposition "attachment; filename*=UTF-8''$arg_filename";
```

Comment:
Some of this configuration comes from 'upstream', such as https://github.com/ONLYOFFICE/document-server-package/blob/master/common/documentserver/nginx/includes/ds-docservice.conf.m4

I can't trace this particular line to that, though, so this seems like a safe change. Changing any of the directives that come from 'upstream' might require some more thought - though possibly still OK, just a matter of documenting the provenance better.

Reply:
It is from here https://github.com/ONLYOFFICE/document-server-package/blob/master/common/documentserver/nginx/includes/ds-docservice.conf.m4#L44

but we mainly change it to fix the gixy error